### ISSUE TITLE
Update Repo links

### ISSUE THREAD

**Issue by obstschale** [nrempel/adonis-queue#1]
I found this package on npmjs.com. I just want to mention that In your `package.json` the repos links are old and not updated. See https://github.com/nrempel/adonis-queue/search?utf8=%E2%9C%93&q=https%3A%2F%2Fgithub.com%2Fnrempel%2Fadonis-bull

**Comment by nrempel** [comment]
ah, thanks @obstschale.

I actually ran into a couple of issues with bull when working with this provider. Some more discussion on that here: https://github.com/nrempel/adonis-kue/issues/1

Basically https://github.com/nrempel/adonis-kue is stable and I'm currently using it in production. This provider is nearly the same but is uses https://github.com/OptimalBits/bull instead of https://github.com/Automattic/kue. I ran into a bug with `bull` when building this and haven't had the time to look into it further.

I'll update those lines when I get a sec.
